lib/src/main/java/me/onebone/toolbar/CollapsingToolbarScaffold.kt

```diff
@@ -75,14 +151,16 @@ fun CollapsingToolbarScaffold(
 	modifier: Modifier,
 	state: CollapsingToolbarScaffoldState,
 	scrollStrategy: ScrollStrategy,
+	snapStrategy: SnapStrategy? = null,
 	toolbarModifier: Modifier = Modifier,
 	toolbar: @Composable CollapsingToolbarScope.() -> Unit,
 	body: @Composable () -> Unit
 ) {
 	val flingBehavior = ScrollableDefaults.flingBehavior()
 
 	val nestedScrollConnection = remember(scrollStrategy, state) {
-		scrollStrategy.create(state.offsetYState, state.toolbarState, flingBehavior)
+		// TODO by RareScrap: Should we make offsetYState public and pass just state?
```

Comment:
Thank you for pointing this, I think having reference to the same `MutableState<T>` object in many different sites is error-prone and should be removed. Now you have added `expandOffset` and `collapseOffset`, they might be removed in favor of these methods.